### ISSUE TITLE
Aparentemente está dando erro

### ISSUE THREAD

**Issue by fpgabriel** [new92/IGFI#52]
![image](https://github.com/new92/IGFI/assets/141476365/9046a396-db1e-46a6-9024-4759777cd755)

**Comment by new92** [comment]
Hey @fpgabriel did you follow the [installation process](https://github.com/new92/IGFI?tab=readme-ov-file#installation-) ? Make sure you executed the `cookies.py` script before the `igfi.py` one. This error typically occurs when you don't provide a session file.

**Comment by fpgabriel** [comment]
![image](https://github.com/new92/IGFI/assets/141476365/edc71f94-7a4a-4f95-85ad-9054992db558)

o que pode ser isso?

**Comment by fpgabriel** [comment]
tenho a versao mais atualizada de python

**Comment by new92** [comment]
Hey @fpgabriel sorry I can't understand it, can you translate it to english ?

**Comment by fpgabriel** [comment]
python3 ./cookies.py does not work

**Comment by new92** [comment]
Yeah yeah, can you also send me the complete error message to better understand it ?

**Comment by fpgabriel** [comment]
![image](https://github.com/new92/IGFI/assets/141476365/865a172b-0869-493f-a288-6b35b82daa75)

**Comment by new92** [comment]
Ok so you need to install the requirements first. To do that enter the following command:
`pip install -r ./files/requirements.txt`. Then enter the `python3 cookies.py` and lastly the `python3 igfi.py` command. But in order not to encounter other errors I would suggest you to follow [the installation process](https://github.com/new92/IGFI/tree/main?tab=readme-ov-file#windows-). Let me know if the issue is resolved when you're done 👍

**Comment by fpgabriel** [comment]
![image](https://github.com/new92/IGFI/assets/141476365/23da4011-5c38-46a1-ae59-b873635cbf94)

**Comment by new92** [comment]
Ok, try this command:
`pip install -r ./files/requirements.txt --target=c:/users/gabri/appdata/local/programs/python/python312/scripts` if that doesn't work try this one:
`pip install -r ./files/requirements.txt --target=c:/users/gabri/appdata/local/programs/python/python312/lib/site-packages`

**Comment by fpgabriel** [comment]
continues with the same error, even after your commands.

I will pay you for your help

**Comment by new92** [comment]
No, no don't worry about it I don't want you to pay me I'll help you for free but a star in the repo would be really helpful ! So back to the script, try with a virtual environment. Like so:
```bash
git clone https://github.com/new92/IGFI
cd IGFI
python -m venv venv
.\venv\Scripts\activate
pip install -r ./files/requirements.txt
python3 ./cookies.py
python3 ./igfi.py -u <username> -p <password> --session <session_file>
```

**Comment by fpgabriel** [comment]
![image](https://github.com/new92/IGFI/assets/141476365/f2c7758f-c9db-436f-9038-607486f0dc72)
Even with everything updated, the same error remains

**Comment by new92** [comment]
@fpgabriel check also [this one on reddit](https://www.reddit.com/r/learnpython/comments/osvtik/python_cant_find_module_even_after_pip_install/) maybe it'll help you.

**Comment by fpgabriel** [comment]
Nothing worked, can we talk another way?

I would like to hire you for a project, but of course, if you are available!

**Comment by new92** [comment]
Hey @fpgabriel yeah you can email me at: new92github@gmail.com

**Comment by fpgabriel** [comment]
I sent an e-mail
